TCAT/Utilities/Styles.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Styles.swift
+//  TCAT
+//
+//  Created by Matt Barker on 10/12/19.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Colors
+
+// MARK: - Fonts
+
+/// Font identifiers
+struct Fonts {
+    
+    struct SanFrancisco {
+        
+        // Pro Display (New)
+//        static let regular = "SFProDisplay-Regular"
```

Comment:
delete if we won't use these

Reply:
We *may* use it... not sure yet. Should I delete and store this locally until we know, or can it just stay in here for now? It won't stay like this for long